.github/workflows/scanner-update-nvd.yaml

```diff
@@ -0,0 +1,38 @@
+name: Fetch and Update NVD CVEs
+
+on:
+  push:
+    branches:
+      - yli3/**
+  schedule:
+    - cron: '0 0 * * *'  # this runs every day at midnight
```

Comment:
The JSON feeds were only updated once per day, but the API data is updated whenever the data is available (I saw this in the docs somewhere, but I unfortunately forgot where). Because of this, I think we should do it more often than just once per day (unless this is just for testing purposes?). I'm thinking maybe once every 4 or 6 hours?

Does this job also have a way for us to run it manually?